impl/etcd/queue/etcd.go

```diff
@@ -163,7 +163,9 @@ func processEntry(ctx context.Context, txn ctxn.Txn, cbs callbacks, dataKV kv) e
 	if dataKV.AdvanceEpoch {
 		return cbs.advanceFunc(ctx, txn)
 	}
-	return cbs.processFunc(ctx, txn, dataKV.Key, dataKV.Val)
+	// Sequence number is automatically added. However, something, e.g., has
```

Comment:
I don't like this neither, but this is the only way I was able to make the API consistent for both external and internal implementations.